lib/stdlib/python/unittest/__main__.py

```diff
@@ -0,0 +1,16 @@
+"""Main entry point"""
+
+import sys
+
+# PyPy.js doesn't have an "argv" so we'll need to set it.
```

Comment:
Should we stub one in as part of the VM setup?  Not sure what we'd put in it though.